Gemfile.lock

```diff
@@ -211,21 +211,21 @@ GEM
       thor (>= 0.14, < 2.0)
     json (2.3.0)
     jsonapi-renderer (0.2.2)
-    kaminari (1.1.1)
+    kaminari (1.2.0)
       activesupport (>= 4.1.0)
-      kaminari-actionview (= 1.1.1)
-      kaminari-activerecord (= 1.1.1)
-      kaminari-core (= 1.1.1)
-    kaminari-actionview (1.1.1)
+      kaminari-actionview (= 1.2.0)
+      kaminari-activerecord (= 1.2.0)
+      kaminari-core (= 1.2.0)
+    kaminari-actionview (1.2.0)
       actionview
-      kaminari-core (= 1.1.1)
-    kaminari-activerecord (1.1.1)
+      kaminari-core (= 1.2.0)
+    kaminari-activerecord (1.2.0)
       activerecord
-      kaminari-core (= 1.1.1)
-    kaminari-core (1.1.1)
+      kaminari-core (= 1.2.0)
+    kaminari-core (1.2.0)
     kwalify (0.7.2)
-    launchy (2.4.3)
-      addressable (~> 2.3)
+    launchy (2.5.0)
```

Comment:
### [launchy](https://github.com/copiousfreetime/launchy)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.4.3 → 2.5.0

[_[change-log](https://github.com/copiousfreetime/launchy/blob/master/README.md), [source-code](https://github.com/copiousfreetime/launchy)_]





<details>
<summary>Commits</summary>

A change of **43** commits. See the full changes on [the compare page](https://github.com/copiousfreetime/launchy/compare/v2.4.3...v2.5.0).

These are the first 10 commits:
- (c66a3ec) [update fixme](https://github.com/copiousfreetime/launchy/commit/c66a3eccd24ccef5593841d7e0566dca87a71342)
- (4c387b8) [updating dependencies](https://github.com/copiousfreetime/launchy/commit/4c387b8d5db9e9d87bbf2a4b6b577a9061f52e24)
- (51ebb6d) [adding Gemfile and gemspec](https://github.com/copiousfreetime/launchy/commit/51ebb6d5810586552da10ce6e062ce3995cf82a8)
- (2edacd0) [adding back .travis.yml](https://github.com/copiousfreetime/launchy/commit/2edacd0112d5f91091c26b0cf1af673d3140fd76)
- (2b01986) [Test on Ruby 2.x patched version](https://github.com/copiousfreetime/launchy/commit/2b0198699495bb5c8366a05c77e77a9199810d68)
- (0bbd9bf) [Merge pull request #100 from nicolasleger/patch-1](https://github.com/copiousfreetime/launchy/commit/0bbd9bfa0b724beb25f2df40ba2820f151e3f075)
- (dcf64e7) [ignore .gem directory](https://github.com/copiousfreetime/launchy/commit/dcf64e7cd849aa1d6133f85efa71be74575c00dd)
- (97eecfd) [Require that the `kfmclient` executable also exist](https://github.com/copiousfreetime/launchy/commit/97eecfdea6f65f56a2bed288b25058b4f5533b33)
- (30fed3d) [Ensure we are passing a String to Addressable](https://github.com/copiousfreetime/launchy/commit/30fed3da140b475179fdf515d7e9304472b44e79)
- (bc36c55) [Add links and lynx fallback console browsers](https://github.com/copiousfreetime/launchy/commit/bc36c559c62cb832979974c36914bb966e3a5fe4)


</details>